src/app/shared/search/search.component.html

```diff
@@ -0,0 +1,7 @@
+<div>
+  <input #searhcInput
+    type="text"
+    [value]="searchValue"
```

Comment:
The task was:
3.	Search control with input and action button.
Place input field, button find and add course button.
Bind [()] input field with component property.
On Find click - console.log input value

Please, use two-way binding as mensioned in the task.

Reply:
Changed